app/views/GoodsRecordsSearchResultView.scala.html

```diff
@@ -84,7 +84,7 @@ <h1 class="govuk-heading-l">@messages("goodsRecordSearchResult.h1", totalRecords
                                 text2 = messages("site.remove"),
                                 visuallyHiddenContent1 = messages("goodsRecords.hidden.change"),
                                 visuallyHiddenContent2 = messages("goodsRecords.hidden.remove"),
-                                href1 = routes.SingleRecordController.onPageLoad(goodsRecord.recordId).url,
+                                href1 = routes.ReviewReasonController.onPageLoad(goodsRecord.recordId).url,
```

Comment:
if this needs to be wherever the single page pops up might it be better if this was on the single goods record onPageLoad, which redirects to this page if necessary

this just means other links will also show it - like when you have just created a good but don't want to categorise it yet

Reply:
I agree, any redirect logic should be there. On it now

Reply:
Unfortunately, I've had to undo this change. The issue is the only way to tell if we need to show the review page is through the toReview field... If we send them to Single record and decide whether to show it there, it would end up in an infinite loop on showing the review page.

Reply:
It will also only be needed here...